src/tests/ftest/server/metadata.py

```diff
@@ -399,6 +399,10 @@ def test_container_removal_after_der_nospace(self):
                     self.log.info(traceback.format_exc())
                     self.log.info("(1.4)No space error shown,  test passed.")
                 break
+        if cont == additional_container - 1:
+            self.fail(
+                "##Storage resource did not exhausted after %d containers",
+                "created", len(container_array))
```

Comment:
```suggestion
            self.fail(
                "##Storage resource was not exhausted after {} containers created".format(len(container_array)))
```
I ran this self.fail line and it didn't work. I think there are too many arguments.

Also, I understand your point, but I think it's better to print what you compared in the if statement in the message. So print something like `"## Resource not exhausted with additional {} containers.".format(additional_container)`

Reply:
Thanks for the catch. 
1. Please see the updated with "{} .format()".
2. I have added print additional cont info at step(1.4) before the break occurred, this will provide with the additional_container information.

Reply:
hmm, it still didn't work on my environment. I ran your latest code:
```
self.fail(
    "##Storage resource did not exhaused after {} containers",
    "created".format(len(container_array)))
```
and got:
TypeError: fail() takes at most 2 arguments (3 given)

Reply:
please see the new upadte..